src/blog/2024/01/send-a-file.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Send a File to Node-RED
+subtitle: A guide to sending a csv file to Node-RED and start interacting with it.
+date: 2024-01-03
+authors: ["grey-dziuba"]
+image: /blog/2023/12/images/uns-data-modeling.png
+tags:
+    - posts
+    - flowfuse
+    - how-to
+---
+
+Have you ever needed to send a CSV file to your Node-RED instance? This file can go on to populate a shift schedule, product specifications, or some other configuration file that is used. In this guide, we are leveraging a Python script that runs a Flask application to post the file to a Node-RED instance and organize the data to be sent on or used.
```

Comment:
@gdziuba Is `curl` available on windows? I think using curl is much easier for everyone.

Reply:
Yes, Windows 10 and above since 2018.  Manufacturing uses earlier versions of that though.  I can add it as a third option though.

Reply:
Yes.  Going to include.